### ISSUE TITLE
Subtitle Link Undefined

### ISSUE THREAD

**Issue by BlackDahlia313** [directus/directus#22788]
### Describe the Bug

When setting a related field to the subtitle part of a card, it doesn't show the related string and the link leads to an undefined 404. 

![image](https://github.com/directus/directus/assets/44903742/eb611ebd-8d5b-47eb-b4e8-e67a1a40122f)
![image](https://github.com/directus/directus/assets/44903742/809f340f-7a30-49fc-8f76-2f1f3a104702)

If you set the subtitle to the related items ID, the list is blank, but the link takes you to the correct item.

### To Reproduce

Go to the card view and add a related field to the subtitle and click to attempt to see it's name.

### Directus Version

10.11.2

### Hosting Strategy

Directus Cloud

**Comment by rijkvanzanten** [comment]
@BlackDahlia313 Am I correctly assuming this is a o2m or m2m using the "Related Values" display?

**Comment by BlackDahlia313** [comment]
@rijkvanzanten Correct! In this instance it is a o2m. With one manufacture to many brands for example!

**Comment by rijkvanzanten** [comment]
The display communicates up to the layout what additional nested relational fields to fetch for it to be able to display things like this, so I'm assuming there's some place in the cards layout where that connection is missed. Just to confirm, this collection does work as expected with the table layout @BlackDahlia313? Also, does that same field work as expected when it's in "Title"?

**Comment by BlackDahlia313** [comment]
> The display communicates up to the layout what additional nested relational fields to fetch for it to be able to display things like this, so I'm assuming there's some place in the cards layout where that connection is missed. Just to confirm, this collection does work as expected with the table layout @BlackDahlia313? Also, does that same field work as expected when it's in "Title"?

Works as expected in table view correct!

![image](https://github.com/directus/directus/assets/44903742/434310ee-d421-4f95-9510-263aac47df36)


But does not work in title for card view. It shows the count, but when clicking than shows the blank string and undefined link. But again if setting to ID, while it doesnt show the ID, the link does work!

**Comment by rijkvanzanten** [comment]
> But does not work in title for card view. It shows the count, but when clicking than shows the blank string and undefined link. 

Ah yes that would indeed confirm it's something up in the layout as a whole, rather than a weird one-off in the subtitle field specifically. Thanks for checking!

**Comment by BlackDahlia313** [comment]
<3
